k8s/helm/Chart.yaml

```diff
@@ -22,3 +22,8 @@ version: 0.0.0-replaceme
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
 appVersion: "0.0.0-replaceme"
+
+annotations:
+  # Dependency for the Backup-CRDs.
+  # Allow all versions up to next major version to avoid breaking changes
+  "k8s.cloudogu.com/ces-dependency/k8s-backup-operator-crd": "0.x"
```

Comment:
I don't know if allowing all 0.x versions is a good idea since 0.x can contain breaking changes any time until it becomes 1.0.

Reply:
So you would enforce the hard current version until 1.0?